extensions/objects/models/objects.py

```diff
@@ -435,7 +477,22 @@ class LogicQuestion(BaseObject):
 
     @classmethod
     def normalize(cls, raw):
-        """Validates and normalizes a raw Python object."""
+        """Validates and normalizes a raw Python object.
+
+        Args:
+            raw: *. A Python object to be validated against the schema,
+                normalizing if necessary.
+
+        Returns:
+            dict. The normalized object containing the following key-value
+                pairs:
+                    assumptions: list. The list containing all the assumptions.
```

Comment:
Give more detail: list(str)? list(bool)? Etc.

Ditto below.

Reply:
Done. Please note that it's a recursive kind of structure containing `list(dict)`s.